### PR TITLE
PRMP-1242: Fix missing auth service bug

### PR DESCRIPTION
**Pull request by abbas-khan10**

### CODE REVIEW

File: lambdas/services/search_patient_details_service.py [hunk 1]
```diff
@@ -26,8 +25,7 @@ def __init__(self, user_role, user_ods_code):
 
     def handle_search_patient_request(self, nhs_number):
         try:
-            auth_service = NhsOauthService(self.ssm_service)
-            pds_api_service = get_pds_service()(self.ssm_service, auth_service)
+            pds_api_service = get_pds_service()
```
Comment (MohammadIqbalAD-NHS) [inline lambdas/services/search_patient_details_service.py:28]: For the sake of consistency, could we modify the name here to `pds_service` as set in other parts of the code? (e.g lloyd_george_validator.py line 215)
```suggestion
            pds_service = get_pds_service()
```

File: lambdas/tests/unit/utils/test_utilities.py [hunk 2]
```diff
@@ -34,6 +37,28 @@ def test_decapitalise_keys():
...
+
+
+def test_get_pds_service_returns_real_pds(monkeypatch):
+    monkeypatch.setenv("PDS_FHIR_IS_STUBBED", "False")
+
+    response = get_pds_service()
+
+    assert isinstance(response, PdsApiService)
+
+
```
Comment (MohammadIqbalAD-NHS) [inline lambdas/tests/unit/utils/test_utilities.py:61]: I've noticed that in `lambdas/utils/utilities.py`, `get_pds_service()` checks `PDS_FHIR_IS_STUBBED` against both "False" and "false" values.
Could we modify these tests to also take into account if the boolean value starts with a lowercase? As testing may require us to modify the `PDS_FHIR_IS_STUBBED` value manually on the AWS console, I could see ourselves interchangeably setting uppercase or lowercase values.

File: lambdas/services/search_patient_details_service.py (hunk 1 above)
Comment (AndyFlintNHS) [reply lambdas/services/search_patient_details_service.py:28]: Good shout! I like your thinking, `pds_api_service` is a pretty loaded term, being the name of the unstubbed service.

File: lambdas/tests/unit/utils/test_utilities.py (hunk 2 above)
Comment (AndyFlintNHS) [reply lambdas/tests/unit/utils/test_utilities.py:61]: Sure, will make these parameterized.

Comment (abbas-khan10) [reply lambdas/tests/unit/utils/test_utilities.py:61]: This is a good point actually, the terraform `contains()` which determines this value will set to a lower case `true/false` our primary check should be for a lower case, but having both would be airtight

**Review by MohammadIqbalAD-NHS — APPROVED** [review]

**Review by AndyFlintNHS — APPROVED** [review]
Approving on Abbas' behalf

**Pull request merged**